composer.json

```diff
@@ -40,21 +40,24 @@
         "zendframework/zenddiagnostics": "^1.0"
     },
     "conflict": {
-        "friendsofsymfony/rest-bundle": "<2.1 || >=3.0",
+        "friendsofsymfony/rest-bundle": "<2.1",
         "jms/serializer": "<0.13",
         "sonata-project/admin-bundle": "<3.1",
         "sonata-project/core-bundle": "<3.20"
     },
     "require-dev": {
         "enqueue/amqp-lib": "^0.8",
-        "friendsofsymfony/rest-bundle": "^2.1",
+        "friendsofsymfony/rest-bundle": "^2.6 || ^3.0",
         "guzzlehttp/guzzle": "^3.8",
         "jms/serializer-bundle": "^2.0 || ^3.0",
         "liip/monitor-bundle": "^2.0",
         "nelmio/api-doc-bundle": "^2.4",
+        "sensio/framework-extra-bundle": "^5.5",
```

Comment:
Why is this needed?

Reply:
For smoke tests, oterwise `Error: Class 'Sensio\Bundle\FrameworkExtraBundle\Configuration\Template' not found`